runtime/brooklyn/src/btcbridge.rs

```diff
@@ -0,0 +1,254 @@
+	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = runtime_common::weights::btc_relay::WeightInfo<Runtime>;
+	type ParachainBlocksPerBitcoinBlock = ParachainBlocksPerBitcoinBlock;
+}
+
+impl security::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = runtime_common::weights::security::WeightInfo<Runtime>;
+}
+
+impl interbtc_currency::Config for Runtime {
+	type SignedInner = SignedInner;
+	type SignedFixedPoint = SignedFixedPoint;
+	type UnsignedFixedPoint = UnsignedFixedPoint;
+	type Balance = Balance;
+	type GetNativeCurrencyId = GetNativeCurrencyId;
+	type GetRelayChainCurrencyId = GetRelayChainCurrencyId;
+	type GetWrappedCurrencyId = GetWrappedCurrencyId;
+	type CurrencyConversion = interbtc_currency::CurrencyConvert<Runtime, Oracle, Loans>;
+}
+
+pub type VaultRewardsInstance = reward::Instance2;
+type VaultId = primitives::VaultId<AccountId, CurrencyId>;
+
+impl reward::Config<VaultRewardsInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type SignedFixedPoint = SignedFixedPoint;
+	type PoolId = CurrencyId;
+	type StakeId = VaultId;
+	type CurrencyId = CurrencyId;
+	type MaxRewardCurrencies = ConstU32<2>;
+}
+
+pub type VaultCapacityInstance = reward::Instance3;
+
+impl reward::Config<VaultCapacityInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type SignedFixedPoint = SignedFixedPoint;
+	type PoolId = ();
+	type StakeId = CurrencyId;
+	type CurrencyId = CurrencyId;
+	type MaxRewardCurrencies = ConstU32<2>;
+}
+
+impl staking::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type SignedFixedPoint = SignedFixedPoint;
+	type SignedInner = SignedInner;
+	type CurrencyId = CurrencyId;
+	type GetNativeCurrencyId = GetNativeCurrencyId;
+}
+
+parameter_types! {
+  pub const MaxExpectedValue: UnsignedFixedPoint = UnsignedFixedPoint::from_inner(<UnsignedFixedPoint as FixedPointNumber>::DIV);
+}
+
+impl fee::Config for Runtime {
+	type FeePalletId = FeePalletId;
+	type WeightInfo = runtime_common::weights::fee::WeightInfo<Runtime>;
+	type SignedFixedPoint = SignedFixedPoint;
+	type SignedInner = SignedInner;
+	type CapacityRewards = VaultCapacity;
+	type VaultRewards = VaultRewards;
+	type VaultStaking = VaultStaking;
+	type OnSweep = interbtc_currency::SweepFunds<Runtime, FeeAccount>;
+	type MaxExpectedValue = MaxExpectedValue;
+	type NominationApi = Nomination;
+}
+
+pub struct BlockNumberToBalance;
+
+impl Convert<BlockNumber, Balance> for BlockNumberToBalance {
+	fn convert(a: BlockNumber) -> Balance {
+		a.into()
+	}
+}
+
+parameter_types! {
+  pub const GetNativeCurrencyId: CurrencyId = runtime_common::constants::currency::NATIVE_CURRENCY_ID;
+  pub const GetRelayChainCurrencyId: CurrencyId = runtime_common::constants::currency::PARENT_CURRENCY_ID;
+  pub const GetWrappedCurrencyId: CurrencyId = runtime_common::constants::currency::WRAPPED_CURRENCY_ID;
+}
+
+// Pallet accounts
+parameter_types! {
+  pub const FeePalletId: PalletId = PalletId(*b"mod/fees");
```

Comment:
is there a sense to add btc prefix ?

Reply:
do you mean the filename start with "btc"?